src/Renderer/MarkerRenderer.php

```diff
@@ -160,7 +160,8 @@ protected function getCoordinates(Item $item): ?LatLng
             $lat = $item->get($latAttribute->getColName());
             $lng = $item->get($lngAttribute->getColName());
 
-            if (!strlen($lat) || !strlen($lng)) {
+
+            if ($lat == '' || $lat === null || $lat == '' || $lat === null) {
```

Comment:
`$lat == ''` does not require an check on `$lat === null`. Please make the comparison strict:

```suggestion
            if ($lat === '' || $lat === null || $lat === '' || $lat === null) {
```